x/evm/types/tracer.go

```diff
@@ -155,20 +150,20 @@ func newTracer(ctx sdk.Context, txHash *common.Hash) (tracer vm.Tracer) {
 		}
 		if traceConfig.Tracer == "" {
 			//Basic tracer with config
-			logConfig := vm.LogConfig{
-				DisableMemory:     traceConfig.DisableMemory,
-				DisableStorage:    traceConfig.DisableStorage,
-				DisableStack:      traceConfig.DisableStack,
-				DisableReturnData: traceConfig.DisableReturnData,
-				Debug:             traceConfig.Debug,
+			logConfig := logger.Config{
+				EnableMemory:     !traceConfig.DisableMemory,
```

Comment:
DisableMemory  changed to. EnableMemory ,means  the memory struct will not return by default .  i suppose the traceConfig will changed at the same time